test/dotnet-publish3.Tests/GivenDotnetPublish3PublishesProjects.cs

```diff
@@ -55,6 +55,12 @@ public void ItPublishesARunnableSelfContainedApp()
             var testProjectDirectory = testInstance.TestRoot;
             var rid = DotnetLegacyRuntimeIdentifiers.InferLegacyRestoreRuntimeIdentifier();
 
+            new Restore3Command()
+                .WithWorkingDirectory(testProjectDirectory)
+                .Execute()
+                .Should()
```

Comment:
When entering fluent assertions, I'm trying to standardize our formatting like this:

````
  foo.Should().Pass()
               .And.HaveReturnCode(0);
````

Having each assertion on a single line makes it much easier to parse the conditions we're testing for. Thoughts?